cal/__init__.py

```diff
@@ -68,21 +69,27 @@ def page_not_found(e):
 def home():
     return render_template('index.html')
 
-
 @app.route("/events/")
 def events():
     now = dt.datetime.now()
-    events = Event.query.filter(Event.start > now)\
-        .filter(Event.start < now + dt.timedelta(weeks=1))
+    events = Event.query.filter(Event.start > now).\
+                         filter(Event.start < now + dt.timedelta(weeks=1))
```

Comment:
I'd prefer this to look like 
```Python
events = Event.query.filter(Event.start > now) \
                    .filter(...)
```

Space between `)` and `\`, and a `.filter` on the second line.